deprecation warning with nodev8

it's a warning only so far in v8; but will need to change sooner or later

```console
> inotify@1.4.2 install /path/to/.../node_modules/inotify
> node-gyp rebuild

make: Entering directory `/path/to/.../node_modules/inotify/build'
  CXX(target) Release/obj.target/inotify/src/bindings.o
  CXX(target) Release/obj.target/inotify/src/node_inotify.o
../src/node_inotify.cc: In function ‘void NodeInotify::InitializeInotify(v8::Handle<v8::Object>)’:
../src/node_inotify.cc:13:60: warning: ‘static v8::Local<v8::ObjectTemplate> v8::ObjectTemplate::New()’ is deprecated (declared at /.../.node-gyp/8.5.0/include/node/v8.h:5611): Use isolate version [-Wdeprecated-declarations]
         Local<ObjectTemplate> global = ObjectTemplate::New();
                                                            ^
  SOLINK_MODULE(target) Release/obj.target/inotify.node
  COPY Release/inotify.node
make: Leaving directory `/.../.node_modules/inotify/build'

+ inotify@1.4.2
added 16 packages in 7.052s
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,6 +3,7 @@ language: node_js
 env:
   - CXX="g++-4.8"
 node_js:
+  - "10"
   - "8"
   - "7"
   - "6"

--- a/src/node_inotify.cc
+++ b/src/node_inotify.cc
@@ -10,7 +10,8 @@ namespace NodeInotify {
         exports->Set(Nan::New<String>("version").ToLocalChecked(),
                     Nan::New<String>(NODE_INOTIFY_VERSION).ToLocalChecked());
 
-        Local<ObjectTemplate> global = ObjectTemplate::New();
+        v8::Isolate* isolate = v8::Isolate::GetCurrent();
+        Local<ObjectTemplate> global = ObjectTemplate::New(isolate);
         Handle<Context> context = Nan::New<Context>(reinterpret_cast<ExtensionConfiguration *>(NULL), global);
         Context::Scope context_scope(context);
 

